includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -395,19 +395,19 @@ public function init_settings () {
 						          self::OPTION_DEFAULT_AVATAR_ROBOHASH   => __( 'Robohash (from Gravatar)', 'anycomment' ),
 					          ],
 				          ] )
-				          ->set_description( esc_html( __( 'Default avatar when user does not have any.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Default avatar when user does not have one.', "anycomment" ) ) ),
 
 				     $this->field_builder()
 				          ->set_id( self::OPTION_REGISTER_DEFAULT_GROUP )
 				          ->select()
-				          ->set_title( __( 'Register User Group', "anycomment" ) )
+				          ->set_title( __( 'Default user role for comment subscribers', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
```suggestion
				          ->set_title( __( 'Default User Role for Comment Subscribers', "anycomment" ) )
```